app/src/main/java/com/example/nickelfoxassignment/assignment0/CalculatorUI.kt

```diff
@@ -13,6 +13,7 @@ class CalculatorUI : AppCompatActivity() {
 
         toolbar = findViewById(R.id.toolbar)
```

Comment:
1. Don't make variables to hold views. id of any view can be used as a view. you'll just have to apply binding in that specific function and use the id of that view.
2. Use View Binding